.github/workflows/pr_check_client_side_changes.yml

```
@@ -27,7 +27,7 @@ jobs:
               repo: context.repo.repo,
               commit_sha: context.sha,
             });
-            const commitHeader = data.message.split('\n')[0];
+            const commitHeader = data.message.split('\n')[0].substring(1); // skip the `#` character
```

```suggestion
            const commitHeader = data.message.split('\n')[0].replace(/#(\d+)/g, 'https://github.com/microsoft/playwright/pull/$1');
```

I think we want this instead, then the change in line 53 can also be removed.